include/uv.h

```diff
@@ -1295,6 +1301,7 @@ enum {
 UV_EXTERN int uv_thread_getpriority(uv_thread_t tid, int* priority);
 UV_EXTERN int uv_thread_setpriority(uv_thread_t tid, int priority);
 
+UV_EXTERN int uv_battery_info(uv_battery_info_t* battery_info);
```

Comment:
Can you document this in docs/src/misc.rst?

Reply:
Will do!